checks/evaluation/signed_releases.go

```diff
@@ -47,11 +47,22 @@ func SignedReleases(name string,
 		f := &findings[i]
 
 		// Debug release name
-		releaseName, err := getReleaseName(f)
-		if err != nil {
-			e := sce.WithMessage(sce.ErrScorecardInternal, "could not get release name")
-			return checker.CreateRuntimeErrorResult(name, e)
+		if f.Outcome == finding.OutcomeNotApplicable {
+			dl.Warn(&checker.LogMessage{
+				Text: "no releases found",
+			})
+			// Generic summary.
+			return checker.CreateInconclusiveResult(name, "no releases found")
+		}
+		releaseName := getReleaseName(f)
+		if releaseName == "" {
+			dl.Warn(&checker.LogMessage{
+				Text: "no releases found",
+			})
+			// Generic summary.
+			return checker.CreateInconclusiveResult(name, "no releases found")
```

Comment:
I actually think this was better as a `CreateRuntimeErrorResult` before, as not having a release name would be an error, and isn't indicative of not having releases

Reply:
👍